src/applications/claims-status/tests/components/ClaimListItemV3.unit.spec.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
+
+import ClaimsListItemV3 from '../../components/ClaimsListItemV3';
+
+describe('<ClaimsListItemV3>', () => {
+  it('should not show any flags', () => {
+    const claim = {
+      id: 1,
+      attributes: {
+        decisionLetterSent: false,
+        developmentLetterSent: false,
+        documentsNeeded: false,
+        status: 'CLAIM_RECEIVED',
+      },
+    };
+
+    const screen = render(<ClaimsListItemV3 claim={claim} />);
+    expect(screen.queryByRole('listitem')).not.to.exist;
+  });
+
+  it('should show closed status', () => {
+    const claim = {
+      id: 1,
+      attributes: {
+        status: 'COMPLETE',
+      },
+    };
+
+    const screen = render(<ClaimsListItemV3 claim={claim} />);
+    expect(screen.getByText('Step 5 of 5: Closed')).to.exist;
+  });
+
+  it('should show the correct status', () => {
+    const claim = {
+      id: 1,
+      attributes: {
+        status: 'INITIAL_REVIEW',
+      },
+    };
+
+    const screen = render(<ClaimsListItemV3 claim={claim} />);
+    expect(screen.getByText(/Initial review/i)).to.exist;
+  });
+
+  it('should show development letter flag', () => {
+    const claim = {
+      id: 1,
+      attributes: {
+        decisionLetterSent: false,
+        developmentLetterSent: true,
+        documentsNeeded: false,
+        status: 'INITIAL_REVIEW',
```

Comment:
This is a nitpick, but I would change the status here because the `developmentLetterSent` flag shouldn't be true when the status is `INITIAL_REVIEW`

Reply:
@jerekshoe so I should set the status to something other than `INITIAL_REVIEW` here?